src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -501,16 +499,16 @@
     short-summary: 'Number of years for which the credentials will be valid. Default: 1 year'
 examples:
   - name: Append a certificate to the service principal with the certificate string.
-    text: az ad sp credential reset --name myapp --cert "MIICoT..." --append
+    text: az ad sp credential reset --id myapp --cert "MIICoT..." --append
```

Comment:
`az ad sp credential list` and `az ad sp credential delete` both use `--id`. Only `az ad sp credential reset` uses `--name`, so we unify them.